CHANGELOG.md

```diff
@@ -84,6 +84,7 @@
 
 ### Tools
 
+* [CHANGE] copyblocks: support for S3 was added as well as the ability to client-side copy between the same or different object storage services. Due to this, the tool now requires `-source-service` and `-destination-service` instead of `-service`. #5486
```

Comment:
[nit] We phrase changelog entries in a different tense (don't ask me what it's called though 😅)

```suggestion
* [CHANGE] copyblocks: add support for S3, as well as the ability to copy between the same or different object storage services. Due to this, the `-source-service` and `-destination-service` flags are now required and the `-service` flag has been removed. #5486
```

Reply:
Searching says present tense imperative. This is something I always flip back and forth on because I can't remember which is the correct style, thanks!

Reply:
I modified this slightly more to
> * [CHANGE] copyblocks: add support for S3 and the ability to copy between different object storage services. Due to this, the `-source-service` and `-destination-service` flags are now required and the `-service` flag has been removed.

Reply:
Looks good, thanks